app/controllers/static_controller.rb

```diff
@@ -0,0 +1,2 @@
+class StaticController < ApplicationController
+end
```

Comment:
- [ ] You need to create an index action here for your root route (`root#index`)